cmd/logs/install/cmd.go

```diff
@@ -212,6 +209,8 @@ func printLog(logs *cmv1.Log) {
 	lines := findNextLines(logs)
 	if lines != "" {
 		fmt.Printf("%s\n", lines)
+	} else {
+		fmt.Printf(".")
```

Comment:
I was thinking instead we could use something like .9 [here](https://godoc.org/github.com/briandowns/spinner) so we'd just have a spinner `... show up within 5 minutes  <spinner>`

Thought @vkareh @jeremyeder ?

Reply:
I like it - will play around with it and push a change